packages/core/src/common/_variables.scss

```diff
@@ -46,11 +55,13 @@ $pt-font-size-large: $pt-grid-size * 1.6 !default;
 $pt-font-size-small: $pt-grid-size * 1.2 !default;
 
 // a little bit extra to ensure the height comes out to just over 18px (and browser rounds to 18px)
-$pt-line-height: math.div($pt-grid-size * 1.8, $pt-font-size) + 0.0001 !default;
```

Comment:
ooc why did you have to drop the `math.div` ?

Reply:
[get-sass-vars](https://github.com/niksy/get-sass-vars) actually interprets and evaluates math for us (kind of surprising when you think about it). when I switched to postcss-simple-vars, we lost that evaluation step. But I think I can actually move back to get-sass-vars; see my other comment